packages/components/src/Form/Fields/FieldInline.tsx

```diff
@@ -45,56 +45,81 @@ interface FieldInlinePropsInternal extends FieldBaseProps {
 const FieldInlineLayout: FC<FieldInlinePropsInternal> = ({
   className,
   children,
+  description,
   detail,
   label,
   id,
   required,
   validationMessage,
-}) => (
-  <label className={className} htmlFor={id}>
-    <InputArea>{children}</InputArea>
-    <Label as="span">
-      <Truncate>{label}</Truncate>
-      {required && <RequiredStar />}
-    </Label>
-    <MessageArea id={`describedby-${id}`}>
-      {detail && (
-        <Paragraph fontSize="xsmall" variant="secondary">
-          {detail}
-        </Paragraph>
-      )}
-      {validationMessage ? <ValidationMessage {...validationMessage} /> : null}
-    </MessageArea>
-  </label>
-)
+}) => {
+  return (
+    <label className={className} htmlFor={id}>
+      {detail && <FieldDetail>{detail}</FieldDetail>}
+      <InputArea>{children}</InputArea>
+      <Label as="span">
+        <Truncate>{label}</Truncate>
+        {required && <RequiredStar />}
+      </Label>
+      <MessageArea id={`${id}-describedby`}>
+        {description && <FieldDescription>{description}</FieldDescription>}
+        {validationMessage ? (
+          <ValidationMessage {...validationMessage} />
+        ) : null}
+      </MessageArea>
+    </label>
+  )
+}
+
+const FieldDetail = styled(Span)`
```

Comment:
This would fix the IE11 issue. I think IE11 doesn't handle inline elements in a grid.
```suggestion
const FieldDetail = styled.div`
```

Reply:
Ah actually `styled(Paragraph)` works better than `styled.div` b/c it includes our default font styles. Updated and pushed!